.devcontainer/ngrx.io/devcontainer.json

```diff
@@ -0,0 +1,59 @@
+// For format details, see https://aka.ms/devcontainer.json. For config options, see the README at:
+// https://github.com/microsoft/vscode-dev-containers/tree/v0.245.2/containers/typescript-node
+{
+  "name": "ngrx.io",
+  "workspaceMount": "type=bind,src=${localWorkspaceFolder},dst=/projects/ngrx.io,consistency=delegated",
```

Comment:
I found the screen, but it appears to not be working.
Putting this PR on draft.

![image](https://user-images.githubusercontent.com/28659384/188327821-3dc82720-ee96-4935-b90b-5e94d334e125.png)